pkgs/by-name/ho/hoarder/package.nix

```diff
@@ -0,0 +1,144 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nodejs,
+  node-gyp,
+  inter,
+  python3,
+  srcOnly,
+  removeReferencesTo,
+  pnpm_9,
+}:
+let
+  version = "0.21.0";
```

Comment:
As you are using `finalAttrs` to get/inherit the version, this could be inlined.